docs/user/commercial/organizations.rst

```diff
@@ -3,13 +3,19 @@ Organizations
 
```

Comment:
We've been making the title of these more verbose generally.. Should we do `Organizations: manage permissions for a set of projects` or something?

Reply:
Good idea, it does "stick" out with its current 1-word title.

"Organizations: permissions for multiple projects and team members" ?

Reply:
Either work for me. though that's maybe a bit long? I'd like to avoid having them take 2 lines in the sidebar if we can.

Reply:
That's a GREAT one :+1:

Reply:
Haha... I just noticed that organizations.rst doesn't at all look like the rest of the headline styles :)

```
Organizations: permissions for projects and teams
-------------------------------------------------

.. include:: /shared/admonition-rtd-business.rst

In this article, we explain how the *organization* feature on Read the Docs allows you to manage access to your projects.
On Read the Docs for Business, your account is linked to an *organization*, whereby you can define both individual and common read and write access to yo>

In this article, we use ACME Corporation as our example organization.

ACME has a few people inside their organization,
some who need full access and some who just need access to one project.

.. seealso::

   :doc:`/guides/manage-read-the-docs-teams`
     A step-by-step guide to managing teams.

Member Types
~~~~~~~~~~~~
```

Reply:
There more I think about Sphinx' "free" or "Laissez-faire" headline syntax, the more insane I think it is.

Like, you can't look at a headline and know what level it's at. You have to look at all the preceding headlines.

And you can't copy-paste from one section to another, because they might use different header syntax.

Reply:
Changed the headline btw :+1:

Reply:
Yea, I much prefer markdown-style sections.